src/common/webrtc/io.rs

```diff
@@ -185,7 +185,7 @@ impl Debug for IoPktChannel {
 
 impl IoPktChannel {
     fn new(tx: smol::channel::Sender<IoPkt>) -> Self {
-        let (txo, rx) = smol::channel::bounded::<IoPkt>(10);
+        let (txo, rx) = smol::channel::bounded::<IoPkt>(25);
```

Comment:
Right, that's roughly what I thought. And, I guess there isn't much need to adjust the value based on whether or not the cache buffer was disabled, since if cache isn't disabled that means we have additional memory so why not 25 then too.